packages/yarnpkg-fslib/sources/ZipFS.ts

```diff
@@ -485,16 +489,21 @@ export class ZipFS extends BasePortableFakeFS {
       },
     });
 
+    const fd = this.openSync(p, ``);
```

Comment:
Why an empty string? Why not `r`?

Reply:
The flag isn't used for anything so I left it out, but meant to fix it... then I forgot to fix it.